aces/Loris
# [Imaging Browser - Data Framework] Make Imaging Browser menu filter use the DataFrameworkMenu

## Brief summary of changes

This updates the imaging browser module to use the DataFrameworkMenu class.

#### Testing instructions (if applicable)

Follow the test plan of the module to make sure there was no change in behaviour
